scipy/sparse/linalg/__init__.py

```diff
@@ -80,7 +80,8 @@
 
    eigs -- Find k eigenvalues and eigenvectors of the square matrix A
    eigsh -- Find k eigenvalues and eigenvectors of a symmetric matrix
-   lobpcg -- Solve symmetric partial eigenproblems with optional preconditioning
+   lobpcg -- Solve symmetric partial eigenproblems with optional
+       preconditioning
```

Comment:
I don't know how to do a proper linebreak here but line is too long for PEP8 and hence my attempt.

Reply:
I basically reverted this change since apparently the tool doesn't care.